libraries/ESP8266WiFi/src/WiFiClientSecureBearSSL.h

```diff
@@ -31,13 +31,12 @@
 
 namespace BearSSL {
 
-class WiFiClientSecure : public WiFiClient {
+class WiFiClientSecureCtx : public WiFiClient {
   public:
-    WiFiClientSecure();
-    WiFiClientSecure(const WiFiClientSecure &rhs);
-    ~WiFiClientSecure() override;
+    WiFiClientSecureCtx();
+    ~WiFiClientSecureCtx() override;
 
-    WiFiClientSecure& operator=(const WiFiClientSecure&) = default; // The shared-ptrs handle themselves automatically
+    WiFiClientSecureCtx& operator=(const WiFiClientSecureCtx&) = delete;
```

Comment:
What's with the `default`->`delete` change?  Intentional?

Reply:
Yes, and I just added `WiFiClientSecure::WiFiClientSecure(const WiFiClientSecure &rhs) = delete;`
It is intentional to forbid copy constructor and operator for the context class.

Reply:
Got it, thx!